src/swagger-specs/sandbox.yaml

```diff
@@ -645,12 +697,14 @@ definitions:
         example: 180e1210-d33a-11e9-8b17-d9e538719ued
       name:
         type: string
+        description: The name of the sandbox you want to delete.
         example: mysandbox
       title:
         type: string
         example: My Sandbox
       state:
         type: string
+        description: Deleting a sandbox updates the sandbox state property to “deleted” and deactivates it. The default production sandbox cannot be deleted.
```

Comment:
I would add some backticks  (state and first mention of deleted), plus remove the word property:
"A successful deletion request a sandbox updates the sandbox `state` to `deleted` and deactivates it. The default production sandbox cannot be deleted."